packages/cubejs-schema-compiler/src/extensions/Funnels.ts

```diff
@@ -41,7 +43,7 @@ ${eventJoin.join('\nLEFT JOIN\n')}
         },
         firstStepConversions: {
           sql: () => 'first_step_user_id',
-          type: 'countDistinct',
+          type: countDistinctType,
```

Comment:
Maybe you can use ternary operator without declaring constant? It seems like you don't use it anywhere.